src/components/UserMenu/index.js

```diff
@@ -33,14 +36,26 @@ const UserMenu = ({ showMenu, setShowMenu }) => {
         </UserMenuButton>
     );
 
+    const firstNameSecondNameCapital = (name) => {
+        if (!name) return;
+
+        const [firstName, lastName] = name.split(' ');
+        if (!lastName) return `${firstName.charAt(0).toUpperCase()}`
+
+        return `${firstName.charAt(0).toUpperCase()}${firstName.slice(1)} ${lastName.charAt(0).toUpperCase()}${lastName.slice(1)}`;
+    }
+
     return (
         <UserMenuContainer>
             {user
                 ? (
                     <>
                         <UserMenuText>
-                            {user.displayName}
+                            {firstNameSecondNameCapital(user.displayName)}
```

Comment:
1. Have in mind you're editing the way the user defines their name in the login provider. We shouldn't care how the user saved their own info. If you see an issue with your own name in tests, you should update your Google/Github profile.
2. This is out of the scope, so should be broken out to a separate PR and a separate ticket.

Reply:
that's correct, i didnt think about it until you said it, it makes alot of sense. i'll remove this part of the code